src/Catalyst.Node.POA.CE.Tests/UnitTests/Config/ConfigCopierTests.cs

```diff
@@ -0,0 +1,144 @@
+#region LICENSE
+
+/**
+* Copyright (c) 2019 Catalyst Network
+*
+* This file is part of Catalyst.Node <https://github.com/catalyst-network/Catalyst.Node>
+*
+* Catalyst.Node is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 2 of the License, or
+* (at your option) any later version.
+*
+* Catalyst.Node is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with Catalyst.Node. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Catalyst.Core.Lib.Config;
+using Catalyst.Protocol.Network;
+using Catalyst.TestUtils;
+using FluentAssertions;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Catalyst.Node.POA.CE.Tests.UnitTests.Config
+{
+    public sealed class ConfigCopierTests : FileSystemBasedTest
+    {
+        public ConfigCopierTests(ITestOutputHelper output) : base(output) { }
+
+        private sealed class ConfigFilesOverwriteTestData : TheoryData<string, NetworkType>
+        {
+            public ConfigFilesOverwriteTestData()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private constructor 'ConfigFilesOverwriteTestData'.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4594394)